### DISCUSSION TITLE
How to add requeset header in WebView2

### DISCUSSION THREAD

**Discussion by EricKwon94** [unoplatform/uno#17460] [Q&A] [answered]
In mobile OS

``` cs
class CoreWebView2NavigationStartingEventArgs
[NotImplemented(new[] { "__ANDROID__", "__IOS__", "IS_UNIT_TESTS", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__" })]
public CoreWebView2HttpRequestHeaders RequestHeaders { get; }
```
NotImplemented.
How to add request header on mobile OS?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. This is indeed not implemented at this time. If you want to set headers, you could integrate a `WKWebView` directly for iOS, and use the `DecidePolicy` set to set them.

If you end up spending time implementing it, you could also make a PR in Uno and contribute support for it!
